composer.json

```diff
@@ -45,10 +45,13 @@
         "apigen": [
             "php devtools/vendor/bin/apigen -c build/config/apigen.neon -d build/api -s src -s vendor/symfony/config -s vendor/symfony/console -s vendor/symfony/yaml -s vendor/guzzle/guzzle/src -s vendor/psr/log --report build/logs/checkstyle-apigen.xml --exclude *Test.php --exclude */Tests/* --google-analytics UA-40398434-1"
         ],
+        "install-box": [
+            "curl -LSs http://box-project.github.io/box2/installer.php | php"
+        ],
         "box": [
             "mkdir -p build/artifacts",
-            "composer update --no-dev --no-interaction",
-            "php -d phar.readonly=false ${BOX_BIN} build"
+            "composer update --no-dev --no-interaction ${COMPOSER_FLAGS}",
```

Comment:
install deps by travis section (it is done already!)
Why are you installing it here second time ?